parmed/formats/sdf.py

```diff
@@ -27,17 +28,11 @@ def id_format(filename):
         is_fmt : bool
             True if it is a sdf file, False otherwise
         """
-        f = genopen(filename, 'r')
+        line = linecache.getline(filename, lineno=4)
         try:
-            for line in f:
-                if not line.strip(): continue
-                if line.startswith('M  END') or line.startswith('$$$$'):
-                    return True
-                else:
-                    continue
+            return line.split()[-1] in ['V2000', 'V3000']
```

Comment:
Can we also test the number of words in that line?  So something like this:

```python
words = line.split()
return len(words) = <blah> and words[-1] in ('V2000', 'V3000')
```
?